python/mxnet/symbol.py

```diff
@@ -504,26 +592,32 @@ def list_auxiliary_states(self):
         return [py_str(sarr[i]) for i in range(size.value)]
 
     def infer_type(self, *args, **kwargs):
-        """Given known types for some arguments, infers the type all arguments
-        and all outputs.
+        """Infers the type of all arguments and all outputs, given the known types
+        for some arguments.
+
+        This function takes the known types of some arguments in either positional way
+        or keyword argument way as input. It returns a tuple of `None` values
+        if there is not enough information to deduce the missing types.
 
-        You can pass in the known types in either positional way or keyword argument way.
-        A tuple of ``None`` values is returned if there is not enough information
-        to deduce the missing types.
         Inconsistencies in the known types will cause an error to be raised.
 
         Example usage:
         ----------
-        >>> a = mxnet.sym.var('a')
-        >>> b = mxnet.sym.var('b')
+        >>> a = mx.sym.var('a')
+        >>> b = mx.sym.var('b')
         >>> c = a + b
-        >>> c.infer_type(a=float32)
-        ([numpy.float32, numpy.float32], [numpy.float32], [])
+        >>> arg_types, out_types, aux_types = c.infer_type(a='float32')
+        >>> arg_types
+        [<type 'numpy.float32'>, <type 'numpy.float32'>]
+        >>> out_types
+        [<type 'numpy.float32'>]
+        >>> aux_types
+        []
 
         Parameters
         ----------
         *args :
-            Provide type of arguments in a positional way.
+            Provide the type of arguments in a positional way.
```

Comment:
This sentence starts with a verb.
Change to "Type of arguments in a positional way."